platform/ios/PolyPodApp/PodApi/PolyOut/FS/PolyOut+FS.swift

```diff
@@ -93,7 +93,7 @@ extension PolyOut {
                     size: try calculateFileSize(filePath.path),
                     time: "\(Int(floor(time.timeIntervalSince1970)))",
                     name: fileStore[url] as? String ?? URL(fileURLWithPath: filePath.path).lastPathComponent,
-                    id: idFromUrl(url: url) ?? ""
+                    id: idFromPodUrl(url: url) ?? ""
```

Comment:
by definition, an id must be unique. If that's not the case, it's not an id.
Again, carried over from before, but as long as we're refactoring, we could maybe think a bit about it; maybe just a change of name would do.

Reply:
It depends what is meant by the id field here. It could mean 2 things:

1. The id of the FileStats object. 
2. The id of the file/folder that we are creating a stat for. 

From what I understand the id here stands for the second option.